docker-compose.yml

```diff
@@ -37,6 +37,7 @@ services:
         image: composer:latest
         volumes:
             - .:/app
+        command: sh -c "tail -f /dev/null"
```

Comment:
why are you using this? if you want to run `composer install/update` you can just `docker run` the image without add a service to do that.

Reply:
resolve with this pr https://github.com/DarkmiraTour/community-event-manager/pull/91/commits/ede4c734d4a57049b507df7e1c431a5793ce6b93